lib/main.dart

```diff
@@ -18,7 +21,10 @@ void main() async {
   await Firebase.initializeApp(
     options: DefaultFirebaseOptions.currentPlatform,
   );
-
+  FlutterError.onError = (details) {
+    FlutterError.presentError(details);
+    if (kReleaseMode) exit(1);
```

Comment:
do we really need this?

Reply:
it does not break anything and if some unhandled errors will occur in release app then it will exit